### PR TITLE
Margarita

### PR DESCRIPTION
**Pull request by am-kantox**
Added `Telemetria` support for `Tempus`:

```erlang
  events: [
    [:tempus, :add],
    [:tempus, :next_busy],
    [:tempus, :next_free],
    [:tempus, :slots, :add],
    [:tempus, :slots, :inverse],
    [:tempus, :slots, :merge]
  ]
```

### CODE REVIEW

File: lib/tempus.ex [hunk 1]
```diff
@@ -208,7 +211,10 @@ defmodule Tempus do
         %Tempus.Slot{from: nil, to: ~U[2020-08-06 23:59:59.999999Z]}
       ]
   """
-  def next_free(%Slots{slots: slots}, opts \\ []),
+  def next_free(slots, opts \\ [])
+
+  @telemetria level: :info
+  def next_free(%Slots{slots: slots}, opts),
```
Comment (facundokantox) [inline lib/tempus.ex:217]: I don't understand this. Can you explain?

**Review by facundokantox — COMMENTED** [review]
All good. Just one small doubt

File: lib/tempus.ex (hunk 1 above)
Comment (am-kantox) [reply lib/tempus.ex:217]: You don’t understand exactly _what_?

Comment (facundokantox) [reply lib/tempus.ex:217]: I've never seen this before. So `def next_free(slots, opts \\ [])` actually calls `def next_free(%Slots{slots: slots}, opts)` without any explicitly stating it. AFAIC see just no `, do:` is a green light 👍

**Review by facundokantox — APPROVED** [review]

File: lib/tempus.ex (hunk 1 above)
Comment (am-kantox) [reply lib/tempus.ex:217]: It’s a so-called “function header.”

It is _mandatory_ when you have [several different clauses with optional parameters](https://stackoverflow.com/a/41357084/2035262).

In this case, I overcome a bug [not yet implemented feature] in `telemetria` that fails to understand default values in the annotated functions. :facepalm: 

I will fix it soon, but meanwhile, this workaround works pretty well.

**Pull request merged**